### DISCUSSION TITLE
Observer for Fraction available after intestinal metabolism

### DISCUSSION THREAD

**Discussion by teutonicod** [Open-Systems-Pharmacology/Forum#877] [Questions & Problems]
Dear all,

I would like to have in PK-Sim the fraction that is absorbed into the system after an oral administration including the intestinal metabolism (I have an enzyme expressed in the intestine). Basically it would be something like the observer "Fraction absorbed into mucosa" but after that the metabolism has happened. 
I tried to use the concentration in the portal vein compartment, but since the concentration change with time I did not get what I expected. 
Could you please give me some suggestions on how to build such observer? Or if you have any other idea on how to do this?

Thanks a lot!

Donato

**Comment by wilbertdew** [comment]
Hi Donato,

The "Fraction absorbed into mucosa" should take into account the intestinal metabolism I believe, and in my simulation it is indeed sensitive to the metabolic rate in the gut.
If this is not what you are looking and I am misunderstanding your post, please let me know.
Best,

Wilbert
